api-reference/upload-file-api/server-side-file-upload.md

```diff
@@ -194,13 +194,15 @@ $imageKit = new ImageKit(
     $url_end_point
 );
 
+
+
 // Upload Image - Binary
-$uploadFile = $imageKit->upload(array(
-    'file' => fopen(__DIR__."/image.jpg", "r"),
-    'fileName' => "my_file_name.jpg",
-    "tags" => implode(",", array("tag1", "tag2")),
-    "customCoordinates" => implode(",", array("10", "10", "100", "100"))
-));
+$uploadFile = $imageKit->uploadFile([
+    "file" => fopen(__DIR__."/image.jpg", "r"),
+    "fileName" => "my_file_name.jpg",
+    "tags" => implode(",", ["tag1", "tag2"]),
```

Comment:
Pass array in tags
Pass normal string in customCoordinates 
```
"tags" => ["tag1", "tag2"],
"customCoordinates" => "10,10,100,100"
```

Same feedback for all similar upload API snippets.